pywr/recorders/_hydropower.pyx

```diff
@@ -164,6 +164,125 @@ cdef class HydropowerRecorder(NumpyArrayNodeRecorder):
 HydropowerRecorder.register()
 
 
+cdef class HydropowerRecorderWithVaribaleTailwater(NumpyArrayNodeRecorder):
```

Comment:
Please can you add some unit tests for the new functionality provided by these components?